### ISSUE TITLE
Typo in 3.3.8 Accessible Authentication, Note 4

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag2ict#408]
### Currently

> There are cases where non-web software has an authentication process and no alternative or assistance mechanism is feasible, for example when entering a password on when starting, powering on / turning on an ICT (device or otherwise). In such situations, it may not be possible for the non-web software to meet this success criterion.

"on when" is wrong

### Proposed

I'm not sure if "on when" needs to change to "or when", "upon", or something else. @maryjom I'm guessing we decided this already in a Task Force resolution, and a word or two was just lost in the transfer. Thanks!

**Comment by ferBonnin** [comment]
looking at the [minutes](https://www.w3.org/2024/05/30-wcag2ict-minutes#t10), this was discussed on the May 30th meeting. The proposal selected was [Option 7 from the Google doc](https://docs.google.com/document/d/18giKt9bddNEgnVmn3f8esr5SGhzJlf6vvX8MyBUmK48/edit#heading=h.ldooitm7d0zk), however, it seems like the google doc didn't have a connector. 

Given that the sentence is listing examples, I propose we use "or" as suggested by @mitchellevan.

**Comment by maryjom** [comment]
From comparing the Google doc with the text in the document, there's an extraneous "on" which needs removal. Must have missed that when putting in the changes we agreed to. Thanks for finding this @mitchellevan and thanks to you @ferBonnin for finding the resolution and Google doc which made it easier to compare.

Would you both agree that removing "on" as shown in PR #442 resolves this issue?

**Comment by maryjom** [comment]
On 18 July, the TF agreed that "on when" should be "when" in the phrase so it reads: 
>There are cases where non-web software has an authentication process and no alternative or assistance mechanism is feasible, for example when entering a password when starting, powering on / turning on an ICT (device or otherwise). In such situations, it may not be possible for the non-web software to meet this success criterion.

**Comment by maryjom** [comment]
Closing this as complete since PR 442 has been merged into the editor's draft.
